package.json

```diff
@@ -45,6 +45,7 @@
     "test": "ng test clr-angular --watch false --code-coverage",
     "test:watch": "ng test clr-angular --watch --code-coverage",
     "test:travis": "npm-run-all test:format test:lint test build build:ks build:dev test:golden",
+    "format:file": "prettier --write",
```

Comment:
I'm not sure if it should be more explicit for documenting the usage.

Users will need to know that `--` is requred to get the list of files appended onto the script. This is an npm convention. 

```npm run format:file -- path/to/file```

Reply:
We need to document this in our build/contributing docs.

Reply:
I added a section to our BUILD notes.